tools/cli/commands.js

```diff
@@ -1190,6 +1190,13 @@ main.registerCommand({
       Console.command("meteor deploy appname"), Console.options({ indent: 2 }));
     return 1;
   }
+  
+  if (process.env.MONGO_URL) {
+    Console.error("meteor reset won't reset the DB provided by MONGO_URL, as it can be dangerous " +
+                  "if you specify MONGO_URL to your production server. If you are sure to do that, " +
+                  "do it yourself by raw mongo queries.");
+    return 1;
```

Comment:
`meteor reset` does other project reset stuff besides Mongo.  Should it  be allowed to do the rest of it's work and just give a `Console.info` warning that it *didn't* clean the database?